index.html

```diff
@@ -258,6 +295,81 @@ <h2>
           different values (or values in a different order).
         </dd>
       </dl>
+      <h3>Receiving new input values</h3>
+      <p>
+        When the system receives new input values from a gamepad, run the following steps:
+      </p>
+      <ol>
+        <li>Let |gamepad:Gamepad| be the {{Gamepad}} instance in |navigator:Navigator|.{{[[gamepads]]}} representing the gamepad.
+        <li>Let |axisValues:sequence&lt;unsigned long&gt;| be the gamepad's <a>ordered array of logical axis input values</a>.
+        <li>Let |buttonValues:sequence&lt;unsigned long&gt;| be the gamepad's <a>ordered array of logical button input values</a>.
+        <li>Queue a task (where?) to <a>update gamepad state</a> for |gamepad| with |axisValues| and |buttonValues|.
+      </ol>
+      To <dfn>update gamepad state</dfn> for |gamepad| with an <a>ordered array of logical axis input values</a> |axisValues:sequence&lt;unsigned long&gt;| and an <a>ordered array of logical button input values</a> |buttonValues:sequence&lt;unsigned long&gt;|, run the following steps:
+      <ol>
+        <li>Let |now:DOMHighResTimeStamp| be a {{DOMHighResTimeStamp}} representing the current time.
+        <li>Let |gamepadsStart:DOMHighResTimeStamp| be the |navigationStart:DOMHighResTimeStamp| attribute of the {{PerformanceTiming}} interface.
```

Comment:
Hmm... this bit is not right for a couple of reasons:  

 1.  `PerformanceTiming` is just an interface (not an instance)
 2. `PerformanceTiming` was not initiated to anything. 
 3. `navigationStart` is "obsolete".

If we just need "when the global object was created", then:  
  
```suggestion
        <li>Let |gamepadsStart| be the [=time origin=] of the [=global object=].
```

Also, we probably don't need an instance of `DOMHighResTimeStamp` above.